kuttl-test.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: kudo.dev/v1alpha1
+kind: TestSuite
+manifestDirs:
+  - ./test/manifests/
+commands:
+  - command: kubectl apply -f https://mesosphere.github.io/kubeaddons/bundle.yaml
+testDirs:
+  - ./addons/traefik/1.7.x/
+startKIND: false
```

Comment:
You're giving a preconfigured kind beforehand to this?

Reply:
Yes. See the definition of KUBECONFIG above.

Reply:
`startKIND: false` is the default and isn't necessary... perhaps you want it to communicate it explicitly.

It is also possible to control the internals kind start config... you could set `kindConfig=test/kind/kubernetes-1.17.5.yaml` here in the kuttl-test.yaml... (with `startKIND: true`) or start kuttl with `kuttl test --kind-config=test/kind/kubernetes-1.17.5.yaml`